packages/components/src/Slider/Slider.component.js

```diff
@@ -26,6 +26,9 @@ function isIconsAvailables(icons) {
  * @param {number} max - maximum value of the slider
  */
 export function getSelectedIconPosition(icons, value, min, max) {
+	if (value === null) {
```

Comment:
Why didn't you check for undefined values ?  With a simple `value == null` you could handle it at the same time.